src/arithmetic/agg_funcs.c

```diff
@@ -9,17 +9,21 @@
 #include "aggregate.h"
 #include "repository.h"
 #include "../value.h"
+#include "../util/arr.h"
 #include "../util/qsort.h"
 #include <assert.h>
 #include <math.h>
 #include "../datatypes/array.h"
+#include "../datatypes/set.h"
 
 
 #define ISLT(a,b) ((*a) < (*b))
 
+#define SI_LT(a,b) (SIValue_Compare((*a), (*b), NULL) > 0)
```

Comment:
Unused, but I think the inequality is also backwards here?

Reply:
@jeffreylovitz I believe you're right.

Reply:
removed